### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anajsana**
- Included "What is a Steering Committee" using the wording from the CHAOSS governance as a reference in their [governing board section](https://chaoss.community/governance/)
- Included non-profit 501 (c)(6) entity term under "Why Linux Foundation" question. Additionally, I improved the wording with concrete examples to clearly state that Developer Relations Foundations follows a community-driven, LF unfunded model, and is NOT an LF-funded model.
- Included new question for those new to OS projects called "What is a Charter" with the charter link and be explicit that charters are living documents and can have multiple versions in the future, as the project evolves

See preview [here](https://github.com/DevRel-Foundation/.github/blob/04fc0f6548b12c48e5bdadb7da7181222159d9d9/profile/README.md)

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -48,12 +48,19 @@ If you have questions that are not answered here, please get in touch via the [D
...
+  In the DevRel Foundation, there is currently a temporary group of volunteers (the Volunteer Steering Committee) who have helped build the initial lightweight processes needed to organize and coordinate the community-driven initiatives.
   
-  This first, non-official Interim SC, composed of tenured Developer Relations professionals, has initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the charter and mission are in place, a new Leadership council will be established. This process will be via elections at a future date to be determined.
+</details>
+<details>
+  <summary>
+    <strong>Who is the current Steering Committee?</strong>
+  </summary><br />
+  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committe participants, are composed of tenured Developer Relations professionals, who initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the   charter and mission are in place, a new leadership will be    established. This process will be via elections at a future date to be determined.
```
Comment (leggetter) [inline profile/README.md:61]:
```suggestion
  There is currently a temporary Volunteer Steering Committee. We, the volunteer Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

File: profile/README.md [hunk 2]
```diff
@@ -48,12 +48,19 @@ If you have questions that are not answered here, please get in touch via the [D
...
 
-  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps forward on building something
+  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps       forward on building something
```
Comment (leggetter) [inline profile/README.md:63]:
```suggestion
  While our profession is relatively small, it is big enough that there are many really good people who can do this work. And we hope they want to! We had the initiative and time to do this, and so we feel it’s reasonable for us, just like anyone else, to take the first steps forward on building something
```

**Review by leggetter — CHANGES_REQUESTED** [review]
A few small change requests

**Comment by anajsana** [comment]
> A few small change requests

thanks! I've just committed your changes and made a few edits re FAQ section:

- Added a few edits on the "what is an SC" using the wording from the CHAOSS governance as a reference in their [governing board section](https://chaoss.community/governance/)

- Added a new question for those new to OS projects called "What is a charter" and be explicit that charters are living documents and can have multiple versions in the future, as the project evolves

**Review by jcleblanc — CHANGES_REQUESTED** [review]
The content lgtm with the exception of some small tweaks to line 66: 
"There is currently a temporary Steering Committee...." as the "temporary" was raised as a confusing point in our last meeting. 

I'd recommend dropped that and instead just stating:
"The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined."

File: profile/README.md [hunk 3]
```diff
@@ -31,27 +31,39 @@ If you have questions that are not answered here, please get in touch via the [D
...
+  </summary><br />
+  The purpose of having a charter for the Developer Relations Foundation is to help people understand its mission and scope. The DevRel Foundation Charter is a living document, allowing the community to propose changes and updates as the project evolves.
+</details>
 <details>
   <summary>
     <strong>Who is the current Steering Committee?</strong>
   </summary><br />
-  
-  This first, non-official Interim SC, composed of tenured Developer Relations professionals, has initiated this “intent to form foundation” built through the LF’s guidance and principles. Once the charter and mission are in place, a new Leadership council will be established. This process will be via elections at a future date to be determined.
+  There is currently a temporary Steering Committee. We, the Steering Committee participants, are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```
Comment (anajsana) [inline profile/README.md:66]:
```suggestion
 The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined.
```

**Comment by anajsana** [comment]
> The content lgtm with the exception of some small tweaks to line 66: 
> 
> "There is currently a temporary Steering Committee...." as the "temporary" was raised as a confusing point in our last meeting. 
> 
> 
> 
> I'd recommend dropped that and instead just stating:
> 
> "The current Steering Committee participants are composed of tenured Developer Relations professionals who initiated this “intent to form foundation” built through the LF’s guidance and principles. A new leadership will be established once the charter and mission are in place. This process will be via elections at a future date to be determined."
> 
> 
> 
> 

Thanks for your review ! I've made the requested changes in line 66 based on your comment 🙂

**Review by jcleblanc — APPROVED** [review]
lgtm; thanks!

**Comment by Cruzn65** [comment]
Thank you. Please remove Temporary or Interim from Github as I think I saw it elsewhere. CheerS!

**Pull request merged**